submissions/Roka20012/pop-up-menu/style.css

```diff
@@ -0,0 +1,204 @@
+.container {
+  width: 266px;
+  height: 411px;
+  position: absolute;
+  display: grid;
+  right: 70px;
+  top: 20px;
+}
+
+.sprite-container {
+  position: relative;
+  padding: 10px;
+  padding-left: 18px;
+  top: 0px;
+  width: 202px;
+  height: 304px;
+  display: none;
+  background-color: #fff;
+  grid-template-columns: 1fr 1fr 1fr;
+  grid-template-rows: 76px 76px 76px;
+}
+
+.container-menu {
+  height: 360px;
+  width: 220px;
+  position: absolute;
+  padding: 10px 30px 0px 10px;
+  top: 40px;
+  border: 1px solid #CCCCCCFF;
+  box-shadow: 0px 0px 10px -2px black;
+  overflow-y: hidden;
+  overflow-x: hidden;
+  display: none;
+}
+
+.mark {
+  border-left: 1px solid #CCCCCCFF;
+  border-top: 1px solid #CCCCCCFF;
+  clip-path: polygon(100% 0%, 0% 100%, 0% 0%);
+  position: absolute;
+  content: "";
+  text-align: center;
+  width: 19px;
+  height: 19px;
+  transform: rotate(45deg);
+  right: 48px;
+  top: 31px;
+  display: none;
+  z-index: 2;
+  background-color: white;
+}
+
+.show-more {
+  display: none;
+}
+
+.check {
+  position: absolute;
+  top: 2px;
+  left: 74%;
+  opacity: 0.6;
+}
+
+.check:hover {
+  opacity: 1;
+}
+
+.check::after {
+  position: absolute;
+  width: 16px;
+  height: 16px;
+  cursor: pointer;
+  background-color: #888;
+  content: "";
+}
+
+.check:focus::after {
+  box-shadow: 0px 0px 1px 1px #A8C9FEFF;
+}
+
+#check-show-more {
+  display: none;
+}
+
+.check:checked ~ #check-show-more {
+  display: grid;
+}
+
+#check-show-more {
+  position: absolute;
+  z-index: 1;
+  top: 367px;
+  left: -3px;
+}
+
+#check-show-more::after {
+  position: absolute;
+  width: 260px;
+  height: 31px;
+  padding-top: 10px;
+  cursor: pointer;
+  content: "Show More";
+  text-align: center;
+  font-family: Arial;
+  font-size: 16px;
+  background-color: #E5E5E5FF;
+  color: #4F4F4FFF;
+}
+
+#check-show-more:hover::after {
+  background-color: #CCC;
+}
+
+#check-show-more:focus::after {
+  box-shadow: 0px 0px 1px 2px #A8C9FEFF;
+}
+
+.check:checked ~ .container-menu .sprite-container,
+.check:checked ~ .mark,
+.check:checked ~ .container-menu .hide-label,
+.check:checked ~ .container-menu {
+  display: grid;
+}
+
+a {
+  display: grid;
+  text-decoration: none;
+  text-align: center;
+  color: black;
+  font-family: Arial;
+  font-size: 14px;
+  margin-top: -7px;
+  margin-bottom: 6px;
+}
+
+a:focus {
+  box-shadow: 0px 0px 1px 1px #A8C9FEFF;
+}
+
+a:hover {
+  box-shadow: 0px 0px 0px 1px #E5E5E5FF;
+}
+
+span {
+  position: relative;
+  top: -10px;
+}
+
+#check-show-more:checked ~ .container-menu {
+  height: 385px;
+  overflow-y: scroll;
+  overflow-x: hidden;
+}
+
+#check-show-more:checked {
+  position: fixed;
+  left: -300px;
+}
+
+.hide,
+.hide-label {
+  display: none;
+}
+
+#check-show-more:checked ~ .container-menu .sprite-container .hide {
```

Comment:
In my opinion, this is a very specific selector. Can be simplified

Reply:
Fix it) Maybe like that? If it does not, please correct me.